From 5bbbde9444765d23bb3df155c70c775d208ac236 Mon Sep 17 00:00:00 2001
From: Michael Krummsdorf <michael.krummsdorf@tqs.de>
Date: Wed, 28 May 2014 14:42:15 +0200
Subject: [PATCH] tqma28x: remove enable_enet_clk in machine file

Let fec driver enable the internal enet_clk driver
when such is defined in device tree.

Therefor allow compatibily to designs with external
enet clk source.

Signed-off-by: Michael Krummsdorf <michael.krummsdorf@tqs.de>
---
 arch/arm/mach-mxs/mach-mxs.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/mach-mxs/mach-mxs.c b/arch/arm/mach-mxs/mach-mxs.c
index 3052d7f..9f2957f 100644
--- a/arch/arm/mach-mxs/mach-mxs.c
+++ b/arch/arm/mach-mxs/mach-mxs.c
@@ -342,7 +342,6 @@ static void __init tqma28_init(void)
 	void __iomem *addr;
 	struct clk *ref_pix, *lcdif_sel;
 
-	enable_clk_enet_out();
 	/* TODO: update_fec_mac_prop(OUI_TQS); */
 	lcdif_sel = clk_get_sys(NULL, "lcdif_sel");
 	ref_pix = clk_get_sys(NULL, "ref_pix");
-- 
1.7.10.4

